rs/sns/swap/tests/common/mod.rs

```diff
@@ -240,9 +240,22 @@ pub fn create_generic_sns_neuron_recipes(count: u64) -> Vec<SnsNeuronRecipe> {
 
 pub fn create_generic_cf_participants(count: u64) -> Vec<CfParticipant> {
     (1..count + 1)
-        .map(|i| CfParticipant {
-            hotkey_principal: i2principal_id_string(i),
-            cf_neurons: vec![CfNeuron::try_new(i, E8).unwrap()],
+        .map(|i| {
+            let i = i * 3;
```

Comment:
Why is this needed?

Reply:
Otherwise some neurons will have controllers that are the hotkeys of other neurons. Now that I think of it, that isn't so bad, but my first thought was just to make them all truly unrelated neurons